### DISCUSSION TITLE
Sensor started by Dexcom receiver - wrong time indication on status page

### DISCUSSION THREAD

**Discussion by Agio56** [NightscoutFoundation/xDrip#2988] [Q&A]
It's just an aesthetic problem. I started the One sensor by Dexcom receiver: on the Home Page the activation time is correct, while on the Classic Status page it is one hour earlier.

**Comment by Navid200** [comment]
This is not something I think we should change.
What is shown on the classic status page is not coming from the transmitter.  It shows when xDrip first connected to the session.
The reason I prefer not to change the value shown on the status page is that it can be useful for those troubleshooting to have this information.

As an example, xDrip does not offer a calibration option on the menu until it has received 3 readings from a session.  If you look at the classic status page and see that xDrip has just started to see this session and needs 2 more readings before you can submit a calibration from xDrip, it may answer a question.  While if you only see on the classic status page the same start time of yesterday, it will be confusing why you cannot calibrate.
There may be other circumstances when you may like to know when xDrip first connected to a session.

I guess we can modify the wording on the classic status page hoping to reduce the confusion.
